### ISSUE TITLE
Historical Data Hovermode

### ISSUE THREAD

**Issue by alexcvasconcellos** [plotly/plotly.js#2605]
I'm not sure if this is already possible, but I'd like to have an option to force the hover event to keep track of the last (most to the left) record on a scattergl plot.

What I mean is: using a hoverdistance of -1, if I move past the midpoint between two records, I get the event for the next record, but I want to have the event fired only after I pass the next value. Is that possible at all?

Thank you and sorry if there's already an issue for that!

**Comment by alexcjohnson** [comment]
And the label as well, I gather? Meaning you want the hover label to always show the closest event to the left of the mouse?

That's not currently possible, no. Would be a great candidate for a community PR 😉 but we should probably agree on an API for it here before anyone starts coding on it.

**Comment by gvwilson** [comment]
Hi - this issue has been sitting for a while, so as part of our effort to tidy up our public repositories I'm going to close it. If it's still a concern, we'd be grateful if you could open a new issue (with a short reproducible example if appropriate) so that we can add it to our stack. Cheers - @gvwilson
